tensorflow/core/util/tensor_bundle/tensor_bundle_test.cc

```diff
@@ -259,6 +278,207 @@ void TestBasic() {
   }
 }
 
+// Type-specific subroutine of SwapBytes test below
+template <typename T>
+void TestByteSwap(const T* forward, const T* swapped, int array_len) {
+  auto bytes_per_elem = sizeof(T);
+
+  // Convert the entire array at once
+  std::unique_ptr<T> forward_copy(new T[array_len]);
```

Comment:
Forgot one more fix I had last time. Should be std::unique_ptr<T[]>. Again somehow this gets exercised internally...

Reply:
Thanks for catching that! I've pushed a fix. It's a bit scary that clang allowed the original code to compile.